### ISSUE TITLE
cli53 incompatible with 2016.09 amazon linux ami.

### ISSUE THREAD

**Issue by balvey** [barnybug/cli53#201]
### cli53 version (cli53 --version)
cli53 version 0.6.9

### OS / Platform
Amazon Linux AMI 2016.09

### Expected behaviour
expecting to create R53 record in hosted zone

### Actual behaviour
gives vague trap error.

### Steps to reproduce
install on amazon linux 2016.09 and try to run rrcreate

Please include full commands and/or bind files that demonstrate the issue, along with the output.

### Was the documentation missing information you required?

Please ensure you've checked: https://github.com/barnybug/cli53/blob/master/README.md

### Could you contribute a fix or help testing with this issue?
I could supply more information or testing if needed. I currently switched back to 2015.09 linux ami and it is working for me.

**Comment by barnybug** [comment]
Could you check you are using the right build for your platform: -386 for 32 bit, -amd64 for 64 bit platform?

**Comment by balvey** [comment]
Sure can. I downloaded cli53 a while ago, don't remember there being different platforms at the time. Is that something new? How can I determine which cli53 I've downloaded?  Here's the md5sum of what I have:
[root@ip-10-50-53-132 ~]# md5sum /usr/bin/cli53
06bc200a0f1f6ed9419d7bbd8d5ebe17  /usr/bin/cli53


The ami's are both 64bit, funny thing is I'm also doing a yum update and bringing the older one up to date on launch. 
The ami it doesn't work on is: amzn-ami-hvm-2016.09.0.20160923-x86_64-gp2 (ami-c481fad3)
The ami it is working with is:  amzn-ami-hvm-2015.09.2.x86_64-gp2 (ami-8fcee4e5)

This may all be for naught, I'm downloading the latest now and trying it out on the newer ami. I'll let you know if that fixes the issues.
